cmd/okctl/create.go

```diff
@@ -250,6 +252,13 @@ and database subnets.`,
 				return formatErr(err)
 			}
 
+			var checkers []servicequota.Checker
+			checkers = append(checkers,
+				servicequota.NewVpcCheck(o.Err, o.CloudProvider, config.DefaultRequiredVpcs),
+				servicequota.NewEipCheck(o.Err, o.CloudProvider, config.DefaultRequiredEpis),
+				servicequota.NewIgwCheck(o.Err, o.CloudProvider, config.DefaultRequiredIgws))
+			servicequota.CheckQuotas(checkers)
```

Comment:
You could do this inline if `CheckQuotas` has a variadic receiver, e.g, `CheckQuotas(checkers ...Checker)`: https://gobyexample.com/variadic-functions

Reply:
Looks better :) Done!